### PR TITLE
spelling: debug...

### PR DESCRIPTION
**Pull request by jsoref**
While these changes technically impact the build, they should be debug only changes that only impact debugging failure cases. There are also only a handful of them which should make review easy.

This is PR 6 of N.

These can definitely be squashed together. I'm slowly peeling them out from other categories...

### CODE REVIEW

File: src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs [hunk 1]
```diff
@@ -255,7 +255,7 @@ private static List<PackageJson> PopulateFromArray(object[] listOfPackages, stri
     public class OSRequirement
     {
         public List<string> architecture { get; set; }
-        public string minimunVersion { get; set; }
+        public string minimumVersion { get; set; }
```
Comment (vors) [inline src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs]: @raghushantha @quoctruong this is technically public API. Are we good with the fix?

File: src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs [hunk 2]
```diff
@@ -342,7 +342,7 @@ public string Source
                     // check whether source is http instead of https
                     if (!Uri.TryCreate(value, UriKind.Absolute, out uri))
                     {
-                        throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Messages.UnsuportedUriFormat, Constants.ProviderName, value));
+                        throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Messages.UnsupportedUriFormat, Constants.ProviderName, value));
```
Comment (vors) [inline src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs]: This should have corresponding fix in the `.resx` file

**Comment by vors** [comment]
I really like these spelling fixes. Thank you @jsoref for doing it!

File: src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs (hunk 1 above)
Comment (jsoref) [inline src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs]: Sorry. I was trying to exclude such changes from this series. I'll kick it out.

File: src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs (hunk 2 above)
Comment (jsoref) [inline src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs]: I'll kick this out too. Sorry.

**Comment by jsoref** [comment]
@vors : I've stripped out those two items.

File: src/Microsoft.PowerShell.PackageManagement/Cmdlets/CmdletBase.cs [hunk 3]
```diff
@@ -484,7 +484,7 @@ protected virtual IHostApi PackageManagementHost
                     PackageVersion = swidObject.Version,
                     PackageProviderName = swidObject.ProviderName,
                     Repository = swidObject.Source,
-                    ExuectionStatus = swidObject.Status,
+                    ExecutionStatus = swidObject.Status,
```
Comment (vors) [inline src/Microsoft.PowerShell.PackageManagement/Cmdlets/CmdletBase.cs]: @raghushantha @quoctruong is it ok to make this change?

Comment (raghushantha) [inline src/Microsoft.PowerShell.PackageManagement/Cmdlets/CmdletBase.cs]: Yes - ok to update. We are not ingesting these events anywhere.

**Comment by vors** [comment]
@jsoref can you open a separate PR for the public filed renaming thing?

**Comment by jsoref** [comment]
@vors, I have three more PRs to produce: local variables, public APIs, and string resource names. I might get the locals up tonight. It's a slow process teasing the three apart.

**Pull request merged**